.github/workflows/global-ci-bundle.yml

```
@@ -593,80 +593,12 @@ jobs:
 
       - name: Run Koncur Kantra tests
-        shell: bash
-        working-directory: koncur
-        env:
-          SKIP_MAVEN: ${{ inputs.koncur_skip_maven }}
-        run: koncur run tests -t kantra -o yaml --output-file test-kantra.yaml
-
-      - name: Upload result yaml
-        uses: actions/upload-artifact@v4
-        if: always()
+        uses: ./koncur-kantra
```

actionlint (1.7.12)

[error] could not parse action metadata in "/home/jailuser/git/koncur-kantra": line 4: unexpected key "type" for definition of input "ref"

(action)